insights/specs/default.py

```diff
@@ -548,6 +548,7 @@ def semid(broker):
     ovirt_engine_ui_log = simple_file("/var/log/ovirt-engine/ui.log")
     ovirt_engine_boot_log = simple_file("/var/log/ovirt-engine/boot.log")
     ovirt_engine_console_log = simple_file("/var/log/ovirt-engine/console.log")
+    ovs_vsctl_list_bridge = simple_command("/usr/bin/ovs-vsctl list bridge")
```

Comment:
@ppandey-redhat  @xiangce this needs a spec for insights_archive as well.

Reply:
Oops, my fault, @ppandey-redhat please raise another PR to add the corresponding `simple_file` for this simple_command defined in the `default.py`

Thank you very much! @bfahr.

Reply:
@bfahr @xiangce - Acknowledged. As per suggestion, I have raised PR https://github.com/RedHatInsights/insights-core/pull/1774 to add spec for `insights_archive`.